app/services/email_service.rb

```diff
@@ -89,7 +89,7 @@ def previous_pom
       }
       return nil if previous.blank?
 
-      StaffMember.new(Prison.new(previous.prison), previous.primary_pom_nomis_id)
+      StaffMember.new(Prison.find_by!(code: previous.prison), previous.primary_pom_nomis_id)
```

Comment:
do we really need a staff member here?

Reply:
Hi Stephen, I have just been trying to refactor this but unfortunately it gets a bit messy when I try to. Unless I am tackling it wrong it feels better to leave it as it is.